src/web/app/pages-help/instructor-help-page/instructor-help-questions-section/instructor-help-questions-section.component.html

```diff
@@ -672,6 +675,12 @@ <h4 *ngIf="displaySubsection(0, 10)">Setting Up Questions</h4>
           The <b>Overall Rank</b> ranks the average rank each recipient receives.
           For example, if recipient A received the ranks <code>(1, 2)</code> and recipient B received the ranks <code>(2, 4, 6)</code>, then recipient A and recipient B's average ranks are 1.5 and 4 respectively. By ranking these two averages, recipient A and B will get an <b>Overall Rank</b> of 1 and 2 respectively.
         </li>
+        <li>
+          The <b>Team Rank</b> works in a similar way to <b>Overall Rank</b> as it uses the average rank received to calculate the rank within the team. to the average rank each recipient receives. However, the average rank of the members are only compared to their own team members.
```

Comment:
Typo in the sentence, I'm assuming the period at the end of "team" was unintended